frontend/packages/kubevirt-plugin/src/components/vms/vm-resource.tsx

```diff
@@ -87,12 +90,12 @@ export const VMDetailsList = ({ vm, vmi, pods, migrations }: VMResourceListProps
 };
 
 type VMResourceSummaryProps = {
-  vm: VMKind;
+  vm: V1VirtualMachine;
 };
 
 type VMResourceListProps = {
-  vm: VMKind;
-  pods?: PodKind[];
-  migrations?: any[];
-  vmi?: VMIKind;
+  vm: V1VirtualMachine;
+  pods?: V1Pod[];
+  migrations?: V1VirtualMachineInstanceMigration[];
```

Comment:
Otherwise it would be quite tedious to create all the new types. 

\+ the whole console & other plugins could benefit from the usual openshift objects